invenio_rdm_records/config.py

```diff
@@ -628,6 +628,9 @@ def lock_edit_published_files(service, identity, record=None, draft=None):
 RDM_DATACITE_FUNDER_IDENTIFIERS_PRIORITY = ("ror", "doi", "grid", "isni", "gnd")
 """Priority of funder identifiers types to be used for DataCite serialization."""
 
+RDM_DATACITE_DUMP_ACCESS_RIGHTS = False
```

Comment:
I wonder if we want to default this to True. I'm not clear on the history of adding and removing this from the serializer...but it generally seems like a good thing to include.

Reply:
I wonder if we want to use a different config variable then? Maybe RDM_DATACITE_DUMP_OPENAIRE_ACCESS_RIGHTS?

I agree the rest can be put in an issue and then we can pick one of those options